config/pre_commit.rb

```diff
@@ -5,6 +5,7 @@
   "\t" => "tab",
   "\r" => "carriage return",
   "\uFEFF" => "byte order marker (BOM)",
+  ("WCA " + "id") => "We prefer 'WCA ID', see https://github.com/thewca/worldcubeassociation.org/issues/268",
```

Comment:
One of the funny things about writing code that checks for illegal strings in your codebase... you have to bend over backwards a bit to make sure that you don't actually type any of those illegal strings when you're writing the checking code!